packages/nimble-components/src/theme-provider/index.ts

```diff
@@ -8,13 +8,29 @@ import { Direction } from '@microsoft/fast-web-utilities';
 import { template } from './template';
 import { styles } from './styles';
 import { Theme } from './types';
+import { documentElementLang } from '../utilities/models/document-element-lang';
+import type { ValidityObject } from '../utilities/models/validator';
 
 declare global {
     interface HTMLElementTagNameMap {
         'nimble-theme-provider': ThemeProvider;
     }
 }
 
+function isInvalidLang(value: string): boolean {
+    try {
+        Intl.DateTimeFormat(value);
```

Comment:
The use of `Intl.DateTimeFormat` here seems a bit arbitrary. Could you pass the value into the constructor of `Intl.Locale` instead, or does `DateTimeFormat` give extra validation? Part of my question stems from the fact that we'll need this in the number column soon as well, so I would hope that any locale that's valid for the date formatter would also be valid for the number formatter. Therefore, it would be nice to write this in a way that wasn't tied to a specific usage of the locale.

Reply:
That works, but I did have to add an eslint suppression (for calling `new` without using the object). For some reason I was able to construct an `Intl.DateTimeFormat` without using `new`, but can't for `Intl.Locale`.